src/status_im/ethereum/json_rpc.cljs

```diff
@@ -183,6 +183,8 @@
    "wallet_getFavourites" {}
    "wallet_deleteCustomToken" {}
    "wallet_getCryptoOnRamps" {}
+   "wallet_getOpenseaCollectionsByOwner" {}
```

Comment:
probably its better to remove Opensea, it might be different provider later

Reply:
I think we can look into a transition if we remove Opensea. This name is more apt.

Currently the shape of our data imitates the shape of Opensea APIs. If we remove it, we can create a new method and get rid of this.